include/llvm/CodeGen/StackMaps.h

```diff
@@ -127,7 +127,10 @@ class StatepointOpers {
   const MachineInstr *MI;
 };
 
-class StackMaps {
+/// StackMapsV1 will be deprecated/deleted when all clients are moved to StackMaps.
```

Comment:
The current plan is to support StackMap v1 and v2 for one release cycle, and deprecate v1 in the next release.